index.js

```diff
@@ -80,15 +89,15 @@ module.exports = inputOptions => {
 
 	if (process.type === 'renderer') {
 		const errorHandler = debounce(error => {
-			handleError('Unhandled Error', error);
+			invokeErrorHandler('Unhandled Error', error);
```

Comment:
So we could add `options.logger(error)` here. @sindresorhus do you have a better approach in mind?